packages/client/modules/email/components/EmailNotifications/EmailKickedOut.tsx

```diff
@@ -0,0 +1,42 @@
+import graphql from 'babel-plugin-relay/macro'
+import {EmailKickedOut_notification$key} from 'parabol-client/__generated__/EmailKickedOut_notification.graphql'
+import React from 'react'
+import {useFragment} from 'react-relay'
+import NotificationTemplate from './EmailNotificationTemplate'
```

Comment:
+1 Using the exported name also for the default import makes it easier to find usages without relying on the ts-node completion.